extensions/wikibase/src/org/openrefine/wikibase/editing/MediaFileUtils.java

```diff
@@ -77,8 +79,32 @@ public void purgePage(long pageid) throws IOException, MediaWikiApiErrorExceptio
         Map<String, String> parameters = new HashMap<>();
         parameters.put("action", "purge");
         parameters.put("pageids", Long.toString(pageid));
-
-        apiConnection.sendJsonRequest("POST", parameters);
+        int retries = maxRetries;
+        long backofTime = maxLagWaitTime;
```

Comment:
`backof` should be `backOff` (note the two "f")

Reply:
Corrected typo